mappings/net/minecraft/world/gen/WorldGenRandom.mapping

```diff
@@ -1,3 +1,18 @@
 CLASS net/minecraft/unmapped/C_rlomrsco net/minecraft/world/gen/WorldGenRandom
-	METHOD setSeed (J)V
+	METHOD m_bggirwfo createRandomDeriver ()Lnet/minecraft/unmapped/C_coqzjjrt;
```

Comment:
Derivative? createRandomDerive? I don't know what this does so im not entirely sure.

Reply:
Changed it to `createPositionalRandomFactory`